proposed/runtime-binding.md

```diff
@@ -0,0 +1,265 @@
+# Runtime Binding Behavior
+
+[Framework-dependent](https://docs.microsoft.com/en-us/dotnet/core/deploying/#framework-dependent-deployments-fdd)  applications require a .NET Core host to find a compatible runtime from a central location. The behavior of runtime binding is critical for both application compatibility and runtime deployment convenience.
+
+This document proposes a model that works for both apps and managed COM components/hosting.
+
+Related content:
+
+* [Roll Forward On No Candidate Fx](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md) doc defines existing behavior. 
+* [Roll forward improvements for COM -- dotnet/core-setup #5062](https://github.com/dotnet/core-setup/issues/5062) specifies a proposal specific to COM. 
+* [--fx-version argument to dotnet exec should apply roll-forward policy -- dotnet/core-setup #4146](https://github.com/dotnet/core-setup/issues/4146) specifies a proposal to enable roll-forward for `--fx-version`
```

Comment:
That is not what dotnet/core-setup#4146 was meant to propose. The issue there is about when a specific fx-version is forced via fx-version, that version should behave the same as if the app had gotten on to it by roll forward.

Reply:
I do like --fx-version composing with --roll-forward in this proposal, it's just orthogonal to that issue. I commented on the issue and changed it's title to be clearer about this.

Reply:
I re-read that issue. I now see that it is unrelated.